apps/ui/src/components/ui/badge.tsx

```diff
@@ -4,6 +4,27 @@ import { cva } from "class-variance-authority";
 
 import { cn } from "@realms-world/utils";
 
+const StatusDot = (status: string | undefined) => {
```

Comment:
Probably best to make a new component for this (generally try to keep the base component library as close to shadcn where possible) - otherwise looking good!

Reply:
removed StatusDot from badge component. I just used the span from the component instead in storybook. StatusDot is now located back in its original place at `apps/nextjs/src/app/_components/StatusDot.tsx`